src/generators/code_gen.py

```diff
@@ -177,6 +177,53 @@ def render(self, visitor: Visitor) -> None:
         visitor.send("}")
 
 
+class SwitchCaseBlock(Block):
+    """
+    Class of representation switch-case construction block.
+
+    Attributes
+    --------
+    expression : ApplyBinOperator
+        expression in switch
+    case_blocks : List[List[Block]]
+        list of blocks in each case
+    cases : List[int]
```

Comment:
Please read [Sphinx docstring style](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html). We can replace Sphinx style with Google style, but it is bad to mix them.

Reply:
Also, types in docstring are redundant and require maintenance

Reply:
Ok, I rework it to Sphinx style